WooCommerce/WooCommerceTests/ViewRelated/Products/Actions Factory/ProductFormActionsFactory+VisibilityTests.swift

```diff
@@ -166,6 +166,35 @@ final class ProductFormActionsFactory_VisibilityTests: XCTestCase {
         // Assert
         XCTAssertFalse(factory.settingsSectionActions().contains(.downloadableFiles(editable: true)))
     }
+
+    // MARK: - Custom fields
+
+    func test_givenExistingProductAndCustomFields_whenCreatingActions_thenCustomFieldsRowIsVisible() {
```

Comment:
Super nit: please use underscores to separate every word except for method names for better readability and consistency with other tests.

Reply:
Good point, added in https://github.com/woocommerce/woocommerce-ios/pull/14068/commits/180303959e8c1ca1a330bd34cd2a0da13ae4a1bb